services/cards-consultation/src/main/java/org/opfab/cards/consultation/controllers/CardOperationsController.java

```diff
@@ -140,6 +140,10 @@ private Flux<String> fetchOldCards0(Instant publishFrom, Instant start, Instant
         return oldCards.map(this::writeValueAsString);
     }
 
+    public void deleteSubscription(String clientId) {
+        cardSubscriptionService.evictSubscription(clientId);
```

Comment:
In the cardSubscriptionService, the subscription is not identify by clientId but with a subId witch is build with method 
CardSubscription.computeSubscriptionId(login,clientId)  

So you need to give user login extract from CurrentUserWithPerimeters

So the actual implementation does not work , it does not find the subscription , see line 1 in the extract from log : 
```
2022-06-14 12:29:26.232  INFO 1 --- [     parallel-1] o.o.c.c.s.CardSubscriptionService        : Subscription with id c11b0965-4216-1d99-f55c-7d09eac3cbfb already evicted , as it is not existing anymore 
2022-06-14 12:29:26.280  INFO 1 --- [or-http-epoll-8] o.o.c.c.services.CardSubscription        : Disposing subscription for user operator1_fr
2022-06-14 12:29:26.280  INFO 1 --- [or-http-epoll-8] o.o.c.c.s.CardSubscriptionService        : Subscription with id operator1_fr#c11b0965-4216-1d99-f55c-7d09eac3cbfb evicted (user operator1_fr)
or1_fr#c11b0965-4216-1d99-f55c-7d09eac3cbfb evicted (user operator1_fr)

 ```

In the log you see the subscription is closed when HTTP connection is closed (line 2)